### DISCUSSION TITLE
Dolphin ffmpegthumbs

### DISCUSSION THREAD

**Discussion by 01fiore** [mocaccinoOS/mocaccino#145] [General] [answered]
Hello, i note that to make the check box for previews of video files in Dolphin appear, the ffmpegthumbs package is missing. 
![Dolphin](https://github.com/user-attachments/assets/3d363dde-5085-4799-a20d-bfed62729870)
sudo luet install apps/ffmpegthumbs
![Dolphin 2](https://github.com/user-attachments/assets/12cc4811-93d1-4151-9ab3-90d0cc09e696)

**Comment by msdobrescu** [comment, marked as answer]
This plugin is available in the community repo as apps/ffmpegthumbs.
